MagicaVoxel-file-format-vox-extension.txt

```diff
@@ -59,13 +63,28 @@ S : Shape Node
  |   |
  S   S
 
+Note: in order to build transformations, you need to traverse the scene
```

Comment:
How to maintain your parsed data shouldnt be part of the doc, would re write this not entirely to just mention that connection between shape nodes and transform nodes and the parenting structure.